build.sbt

```diff
@@ -242,9 +242,9 @@ lazy val `kamon-cats-io` = (project in file("instrumentation/kamon-cats-io"))
       kanelaAgent % "provided",
       {
         if(scalaBinaryVersion.value == "2.11")
-          "org.typelevel" %% "cats-effect" % "2.0.0" % "provided"
+          "org.typelevel" %% "cats-effect" % "3.3.5" % "provided"
```

Comment:
I think it might be saner to split this into kamon-cats-io/kamon-cats-io_3 ?

Reply:
Probably the best way to go indeed.. Just gotta make it work first :wink: